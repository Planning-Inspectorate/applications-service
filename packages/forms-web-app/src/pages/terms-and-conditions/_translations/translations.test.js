const enTermsAndConditionsTranslations = require('./en.json');
const cyTermsAndConditionsTranslations = require('./cy.json');

describe('pages/terms-and-conditions/_translations', () => {
	it('should return the correct English translations', () => {
		expect(enTermsAndConditionsTranslations).toMatchSnapshot();
	});

	it('should return the correct Welsh translations', () => {
		expect(cyTermsAndConditionsTranslations).toMatchSnapshot();
	});

	it('should return the same keys for English and Welsh translations', () => {
		expect(enTermsAndConditionsTranslations).toHaveSameKeys(cyTermsAndConditionsTranslations);
	});
});
