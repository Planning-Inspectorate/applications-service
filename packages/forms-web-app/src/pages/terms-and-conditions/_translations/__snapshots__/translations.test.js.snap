// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages/terms-and-conditions/_translations should return the correct English translations 1`] = `
{
  "changesToTerms": {
    "heading1": "Changes to these terms and conditions",
    "paragraph1": "Please check these terms and conditions regularly. We can update them at any time without notice.",
    "paragraph2": "You'll agree to any changes if you continue to use National Infrastructure Projects after the terms and conditions have been updated.",
    "paragraph3": "Last updated 20 March 2024.",
  },
  "common": {
    "linkText1": "National Infrastructure Projects",
    "linkText2": "National Infrastructure Projects",
    "linkText3": "contact us",
    "linkText4": "Contact us",
  },
  "disclaimer": {
    "heading1": "Disclaimer",
    "listItem1": "current",
    "listItem10": "income or revenue",
    "listItem11": "salary, benefits or other payments",
    "listItem12": "business",
    "listItem13": "profits or contracts",
    "listItem14": "opportunity",
    "listItem15": "anticipated savings",
    "listItem16": "data",
    "listItem17": "goodwill or reputation",
    "listItem18": "tangible property",
    "listItem19": "intangible property, including loss, corruption or damage to data or any computer system",
    "listItem2": "secure",
    "listItem20": "wasted management or office time",
    "listItem21": "death or personal injury arising from our negligence",
    "listItem22": "fraudulent misrepresentation",
    "listItem23": "any other liability which cannot be excluded or limited under applicable law",
    "listItem3": "accurate",
    "listItem4": "complete",
    "listItem5": "free from bugs or viruses",
    "listItem6": "any direct, indirect or consequential losses",
    "listItem7": "any loss or damage caused by civil wrongs ('tort', including negligence), breach of contract or otherwise",
    "listItem8": "the use of {{-link}} and any websites that are linked to or from it",
    "listItem9": "the inability to use {{-link}} and any websites that are linked to or from it",
    "paragraph1": "This section does not reduce or remove our responsibilities or people's rights concerning data protection in any manner.",
    "paragraph2": "While we make every effort to keep {{-link}} up to date, we do not provide any guarantees, conditions or warranties that the information will be:",
    "paragraph3": "We do not publish advice on {{-link}}. You should get professional or specialist advice before doing anything on the basis of the content.",
    "paragraph4": "We're not liable for any loss or damage that may come from using {{-link}}. This includes:",
    "paragraph5": "This applies if the loss or damage was foreseeable, arose in the normal course of things or you advised us that it might happen.",
    "paragraph6": "This includes (but is not limited to) the loss of your:",
    "paragraph7": "We may still be liable for:",
  },
  "general": {
    "heading1": "General",
    "paragraph1": "There may be legal notices elsewhere on {{-link}} that relate to how you use the site.",
    "paragraph2": "We're not liable if we fail to comply with these terms and conditions because of circumstances beyond our reasonable control.",
    "paragraph3": "We might decide not to exercise or enforce any right available to us under these terms and conditions. We can always decide to exercise or enforce that right at a later date.",
    "paragraph4": "Doing this once will not mean we automatically waive the right on any other occasion.",
    "paragraph5": "If any of these terms and conditions are held to be invalid, unenforceable or illegal for any reason, the remaining terms and conditions will still apply.",
  },
  "governingLaw": {
    "heading1": "Governing law",
    "paragraph1": "These terms and conditions are governed by and construed in accordance with the laws of England and Wales.",
    "paragraph2": "Any dispute you have which relates to these terms and conditions, or your use of {{-link}} (whether it be contractual or non-contractual), will be subject to the exclusive jurisdiction of the courts of England and Wales.",
  },
  "informationAboutYou": {
    "heading1": "Information about you and your visits to National Infrastructure Projects",
    "paragraph1": "By using National Infrastructure Projects, you agree to us collecting this information and confirm that any data you provide is accurate.",
    "phrase1": "We collect information about you in accordance with our {{-link}}",
    "phrase1LinkText": "privacy policy",
    "phrase2": "and our {{-link}}.",
    "phrase2LinkText": "cookie policy",
  },
  "introduction": {
    "heading1": "Terms and conditions",
    "paragraph1": "This page and any pages it links to explains {{-link}} terms of use.",
    "paragraph1LinkText": "National Infrastructure Project's",
    "paragraph2": "You must agree to these to use {{-link}}.",
  },
  "linkingFromNIP": {
    "heading1": "Linking from National Infrastructure Projects",
    "listItem1": "the protection of any information you give to these websites",
    "listItem2": "any loss or damage that may come from your use of these websites, or any other websites they link to",
    "paragraph1": "{{-link}} links to websites that are managed by other government departments and agencies, service providers or other organisations. We do not have any control over the content on these websites.",
    "paragraph2": "We're not responsible for:",
    "paragraph3": "You agree to release us from any claims or disputes that may come from using these websites.",
    "paragraph4": "You should read all terms and conditions, privacy policies and end user licences that relate to these websites before you use them.",
  },
  "linkingToNIP": {
    "heading1": "Linking to National Infrastructure Projects",
    "listItem1": "charge your website's users to click on a link to any page on {{-link}}",
    "listItem2": "say your website is associated with or endorsed by {{-link}} or another government department or agency",
    "listItem2LinkText": "National Infrastructure Projects",
    "paragraph1": "We welcome and encourage other websites to link to {{-link}}.",
    "paragraph2": "However we will not give permission to:",
  },
  "offences": {
    "heading1": "Viruses, hacking and other offences",
    "paragraph1": "When using {{-link}}, you must not introduce viruses, trojans, worms, logic bombs or any other material that's malicious or technologically harmful.",
    "paragraph2": "You must not try to gain unauthorised access to {{-link}}, the server on which it's stored or any server, computer or database connected to it.",
    "paragraph3": "You must not attack {{-link}} in any way. This includes denial-of-service attacks.",
    "paragraph4": "We'll report any attacks or attempts to gain unauthorised access to {{-link}} to the relevant law enforcement authorities and share information about you with them.",
  },
  "requestsToRemoveContent": {
    "heading1": "Requests to remove content",
    "listItem1": "in order to comply with data protection legislation covering the rights and freedoms of individuals",
    "listItem2": "if it breaches copyright laws, contains sensitive personal data or material that may be considered obscene or defamatory",
    "paragraph1": "You can ask for content to be removed from {{-link}}. Please note the National Infrastructure process is governed by the principle of openness. If you request for content to be removed, in the interest of fairness, it will usually not be taken into consideration by the Examining Authority.",
    "paragraph2": "We'll remove content:",
    "paragraph3": "{{-link}} to ask for content to be removed. You'll need to send us the web address (URL) of the content and explain why you think it should be removed. We'll reply to let you know whether we'll remove it.",
    "paragraph4": "We remove content at our discretion in discussion with the department or agency responsible for it. You can still request information under the {{-link}}",
    "paragraph4LinkText": "Freedom of Information Act",
    "phrase1": "and the {{-link}}.",
    "phrase1LinkText": "Data Protection Act",
  },
  "servicesAndTransactions": {
    "heading1": "Services and transactions",
    "paragraph1": "You can use {{-link}} to access online government services and transactions, for example 'register to have your say'.",
  },
  "usingNIP": {
    "heading1": "Using National Infrastructure Projects",
    "paragraph1": "You agree to use {{-link}} only for lawful purposes. You must also use it in a way that does not infringe the rights of, or restrict or inhibit the use and enjoyment of, this site by anyone else.",
    "paragraph2": "We update {{-link}} all the time. We can change or remove content at any time without notice.",
  },
  "usingNIPContent": {
    "heading1": "Using National Infrastructure Projects content",
    "paragraph1": "Some content is exempt from the OGL - check the {{-link}}.",
    "paragraph10": "The most up to date version of our content will always be on {{-link}}.",
    "paragraph1LinkText": "list of exemptions",
    "paragraph2": "Departmental logos and crests are also exempt from the OGL, except when they form an integral part of a document or dataset.",
    "paragraph3": "If any content is not subject to Crown copyright protection or published under the OGL, we'll usually credit the author or copyright holder.",
    "paragraph4": "We do not authorise the use or republishing of third party documents without the permission of the original intellectual property holder.",
    "paragraph5": "This covers all material not owned by {{-link}} including those created by applicants, interested parties or interested groups.",
    "paragraph6": "If you have questions about republishing any content on our service please {{-link}}.",
    "paragraph7": "{{-link}} if you want to reproduce a piece of content but are not sure if it's covered by Crown copyright or the OGL.",
    "paragraph8": "We make most of the content on {{-link}} available through feeds for other websites and applications to use. The websites and applications that use our feeds are not our products, and they might use versions of our content that have been edited and stored for later use ('cached').",
    "paragraph9": "We do not give any guarantees, conditions or warranties about the accuracy or completeness of any content used by these products. We're not liable for any loss or damage that may come from your use of these products.",
    "phrase1": "Most content on {{-link}}",
    "phrase2": "is subject to {{-link}}",
    "phrase2LinkText": "Crown copyright protection",
    "phrase3": "and is published under the {{-link}} (OGL).",
    "phrase3LinkText": "Open Government License",
  },
  "virusProtection": {
    "heading1": "Virus protection",
    "paragraph1": "We make every effort to check and test {{-link}} for viruses at every stage of production.",
    "paragraph2": "You must make sure that the way you use {{-link}} does not expose you to the risk of viruses, malicious computer code or other forms of interference which can damage your computer system.",
    "paragraph3": "We're not responsible for any loss, disruption or damage to your data or computer system that might happen when you use {{-link}}.",
  },
  "whoWeAre": {
    "heading1": "Who we are",
    "paragraph1": "{{-link}} are operated by the Planning Inspectorate.",
    "paragraph2": "The Planning Inspectorate is a government agency.",
    "paragraph3": "If you have questions about our service please {{-link}}.",
  },
}
`;

exports[`pages/terms-and-conditions/_translations should return the correct Welsh translations 1`] = `
{
  "changesToTerms": {
    "heading1": "Newidiadau i'r telerau ac amodau hyn",
    "paragraph1": "Gwiriwch y telerau ac amodau hyn yn rheolaidd. Gallwn eu diweddaru unrhyw bryd heb roi rhybudd.",
    "paragraph2": "Byddwch yn cytuno ag unrhyw newidiadau os byddwch yn parhau i ddefnyddio Prosiectau Seilwaith Cenedlaethol ar ôl i'r telerau ac amodau gael eu diweddaru.",
    "paragraph3": "Diweddarwyd ddiwethaf 20 Mawrth 2024.",
  },
  "common": {
    "linkText1": "Prosiectau Seilwaith Cenedlaethol",
    "linkText2": "Brosiectau Seilwaith Cenedlaethol",
    "linkText3": "cysylltwch â ni",
    "linkText4": "Cysylltwch â ni",
  },
  "disclaimer": {
    "heading1": "Ymwadiad",
    "listItem1": "yn gyfredol",
    "listItem10": "incwm neu refeniw",
    "listItem11": "cyflog, budd-daliadau neu daliadau eraill",
    "listItem12": "busnes",
    "listItem13": "elw neu gontractau",
    "listItem14": "cyfle",
    "listItem15": "arbedion disgwyliedig",
    "listItem16": "data",
    "listItem17": "ewyllys da neu enw da",
    "listItem18": "eiddo diriaethol",
    "listItem19": "eiddo anniriaethol, gan gynnwys colli, llygru neu ddifrodi data neu unrhyw system gyfrifiadurol",
    "listItem2": "yn ddiogel",
    "listItem20": "gwastraffu amser rheoli neu swyddfa",
    "listItem21": "marwolaeth neu anaf personol sy'n deillio o'n hesgeuluster",
    "listItem22": "camliwio twyllodrus",
    "listItem23": "unrhyw atebolrwydd arall na ellir ei eithrio neu ei gyfyngu o dan y gyfraith gymwys",
    "listItem3": "yn gywir",
    "listItem4": "yn gyflawn",
    "listItem5": "yn rhydd rhag bygiau neu feirysau",
    "listItem6": "unrhyw golledion uniongyrchol, anuniongyrchol neu ganlyniadol",
    "listItem7": "unrhyw golled neu ddifrod a achoswyd gan gamweddau sifil ('camweddau', gan gynnwys esgeuluster), tor contract neu fel arall",
    "listItem8": "defnyddio {{-link}} ac unrhyw wefannau y mae'n cynnwys dolenni iddynt neu wefannau sy'n cynnwys dolenni iddo",
    "listItem9": "anallu i ddefnyddio {{-link}} ac unrhyw wefannau y mae'n cynnwys dolenni iddynt neu wefannau sy'n cynnwys dolenni iddo",
    "paragraph1": "Nid yw'r adran hon yn lleihau nac yn dileu ein cyfrifoldebau na hawliau pobl yn ymwneud â diogelu data mewn unrhyw ffordd.",
    "paragraph2": "Er y gwnawn bob ymdrech i gadw {{-link}} yn gyfredol, nid ydym yn rhoi unrhyw sicrwydd, amodau na gwarantau y bydd y wybodaeth:",
    "paragraph3": "Nid ydym yn cyhoeddi cyngor ar {{-link}}. Dylech gael cyngor proffesiynol neu arbenigol cyn gwneud unrhyw beth ar sail y cynnwys.",
    "paragraph4": "Nid ydym yn atebol am unrhyw golled neu ddifrod a allai ddeillio o ddefnyddio {{-link}}. Mae hyn yn cynnwys:",
    "paragraph5": "Mae hyn yn berthnasol os oedd y golled neu'r difrod yn rhagweladwy, wedi digwydd yn ôl y drefn arferol neu os rhoddoch wybod i ni y gallai ddigwydd.",
    "paragraph6": "Mae hyn yn cynnwys (ymhlith pethau eraill) colli eich:",
    "paragraph7": "Fe allem fod yn atebol am y canlynol o hyd:",
  },
  "general": {
    "heading1": "Cyffredinol",
    "paragraph1": "Fe allai fod hysbysiadau cyfreithiol mewn mannau eraill ar {{-link}} sy'n ymwneud â sut rydych yn defnyddio'r safle.",
    "paragraph2": "Ni fyddwn yn atebol os na fyddwn yn cydymffurfio â'r telerau ac amodau hyn oherwydd amgylchiadau sydd y tu hwnt i'n rheolaeth resymol.",
    "paragraph3": "Efallai y byddwn yn penderfynu peidio ag arfer neu orfodi unrhyw hawl sydd ar gael i ni o dan y telerau ac amodau hyn. Gallwn bob amser benderfynu arfer neu orfodi'r hawl honno'n ddiweddarach.",
    "paragraph4": "Ni fydd gwneud hyn unwaith yn golygu y byddwn yn ildio'r hawl yn awtomatig ar unrhyw achlysur arall.",
    "paragraph5": "Os bernir bod unrhyw un o'r telerau ac amodau hyn yn annilys, yn anorfodadwy neu'n anghyfreithlon am unrhyw reswm, bydd y telerau ac amodau sy'n weddill yn berthnasol o hyd.",
  },
  "governingLaw": {
    "heading1": "Y gyfraith lywodraethol",
    "paragraph1": "Mae'r telerau ac amodau hyn yn cael eu llywodraethu gan ddeddfau Cymru a Lloegr a chânt eu dehongli yn unol â nhw.",
    "paragraph2": "Bydd unrhyw anghydfod sydd gennych sy'n ymwneud â'r telerau ac amodau hyn, neu'ch defnydd o {{-link}} (boed hynny'n gytundebol neu'n anghytundebol), yn ddarostyngedig i awdurdodaeth lwyr llysoedd Cymru a Lloegr.",
  },
  "informationAboutYou": {
    "heading1": "Gwybodaeth amdanoch chi a'ch ymweliadau â Phrosiectau Seilwaith Cenedlaethol",
    "paragraph1": "Trwy ddefnyddio Prosiectau Seilwaith Cenedlaethol, rydych yn cytuno i ni gasglu'r wybodaeth hon ac yn cadarnhau bod unrhyw ddata a roddwch yn gywir.",
    "phrase1": "Rydym yn casglu gwybodaeth amdanoch yn unol a'n {{-link}}",
    "phrase1LinkText": "polisi preifatrwydd",
    "phrase2": "â'n {{-link}}.",
    "phrase2LinkText": "polisi cwcis",
  },
  "introduction": {
    "heading1": "Telerau ac amodau",
    "paragraph1": "Mae'r dudalen hon ac unrhyw dudalennau y mae'n cysylltu â nhw yn esbonio telerau defnydd {{-link}}.",
    "paragraph1LinkText": "Prosiectau Seilwaith Cenedlaethol",
    "paragraph2": "Mae'n rhaid i chi gytuno â'r rhain i ddefnyddio {{-link}}.",
  },
  "linkingFromNIP": {
    "heading1": "Darparu dolenni o Brosiectau Seilwaith Cenedlaethol",
    "listItem1": "ddiogelu unrhyw wybodaeth a roddwch i'r gwefannau hyn",
    "listItem2": "unrhyw golled neu ddifrod a allai ddeillio o'ch defnydd o'r gwefannau hyn, neu unrhyw wefannau eraill maen nhw'n darparu dolenni iddynt",
    "paragraph1": "Mae {{-link}} yn darparu dolenni i wefannau sy'n cael eu rheoli gan adrannau ac asiantaethau eraill y llywodraeth, darparwyr gwasanaethau neu sefydliadau eraill. Nid oes gennym unrhyw reolaeth dros gynnwys y gwefannau hyn.",
    "paragraph2": "Nid ydym yn gyfrifol am:",
    "paragraph3": "Rydych yn cytuno i'n rhyddhau o unrhyw hawliadau neu anghydfodau a allai ddeillio o ddefnyddio'r gwefannau hyn.",
    "paragraph4": "Dylech ddarllen yr holl delerau ac amodau, polisïau preifatrwydd a thrwyddedau defnyddwyr sy'n ymwneud â'r gwefannau hyn cyn i chi eu defnyddio.",
  },
  "linkingToNIP": {
    "heading1": "Darparu dolenni i Brosiectau Seilwaith Cenedlaethol",
    "listItem1": "i godi tâl ar ddefnyddwyr eich gwefan i glicio ar ddolen i unrhyw un o dudalennau {{-link}}",
    "listItem2": "i ddweud bod eich gwefan yn gysylltiedig â {{-link}} neu'n cael ei chefnogi ganddo ef neu unrhyw adran neu asiantaeth arall o'r llywodraeth",
    "listItem2LinkText": "Phrosiectau Seilwaith Cenedlaethol",
    "paragraph1": "Rydym yn croesawu ac yn annog gwefannau eraill i ddarparu dolenni i {{-link}}.",
    "paragraph2": "Fodd bynnag, ni fyddwn yn rhoi caniatâd:",
  },
  "offences": {
    "heading1": "Feirysau, hacio a throseddau eraill",
    "paragraph1": "Pan fyddwch yn defnyddio {{-link}}, mae'n rhaid i chi beidio â chyflwyno feirysau, feirysau Troea, mwydod, bomiau rhesymeg neu unrhyw ddeunydd arall sy'n faleisus neu'n dechnolegol niweidiol.",
    "paragraph2": "Mae'n rhaid i chi beidio â cheisio cael mynediad heb ei awdurdodi at {{-link}}, y gweinydd y mae'n cael ei storio arno nac unrhyw weinydd, cyfrifiadur neu gronfa ddata sy'n gysylltiedig ag ef.",
    "paragraph3": "Mae'n rhaid i chi beidio ag ymosod ar {{-link}} mewn unrhyw ffordd. Mae hyn yn cynnwys ymosodiadau atal gwasanaeth.",
    "paragraph4": "Byddwn yn rhoi gwybod i'r awdurdodau priodol sy'n gorfodi'r gyfraith am unrhyw ymosodiadau neu geisiadau i gael mynediad heb ei awdurdodi at {{-link}} ac yn rhannu gwybodaeth amdanoch gyda nhw.",
  },
  "requestsToRemoveContent": {
    "heading1": "Ceisiadau i ddileu cynnwys",
    "listItem1": "er mwyn cydymffurfio â deddfwriaeth diogelu data sy'n ymwneud â hawliau a rhyddidau unigolion",
    "listItem2": "os yw'n torri deddfau hawlfraint, yn cynnwys data personol sensitif neu ddeunydd y gellid ystyried ei fod yn anweddus neu'n ddifenwol",
    "paragraph1": "Gallwch ofyn am gael dileu cynnwys o {{-link}}. Sylwch fod y broses Seilwaith Cenedlaethol yn cael ei llywodraethu gan egwyddor bod yn agored. Os byddwch yn gofyn am gael dileu cynnwys, er mwyn tegwch, ni fydd yn cael ei ystyried gan yr Awdurdod Archwilio, fel arfer.",
    "paragraph2": "Byddwn yn dileu cynnwys:",
    "paragraph3": "{{-link}} i ofyn am gael dileu cynnwys. Bydd angen i chi anfon cyfeiriad gwe (URL) y cynnwys atom ac esbonio pam y dylid ei ddileu, yn eich barn chi. Byddwn yn ymateb i roi gwybod i chi p'un a fyddwn yn ei ddileu.",
    "paragraph4": "Byddwn yn dileu cynnwys yn ôl ein disgresiwn mewn trafodaeth â'r adran neu'r asiantaeth sy'n gyfrifol amdano. Gallwch ofyn am wybodaeth o hyd o dan y {{-link}}",
    "paragraph4LinkText": "Ddeddf Rhyddid Gwybodaeth",
    "phrase1": "a'r {{-link}}.",
    "phrase1LinkText": "Ddeddf Diogelu Data",
  },
  "servicesAndTransactions": {
    "heading1": "Gwasanaethau a thrafodion",
    "paragraph1": "Gallwch ddefnyddio {{-link}} i gael mynediad at wasanaethau a thrafodion y llywodraeth, er enghraifft 'cofrestru i leisio'ch barn'.",
  },
  "usingNIP": {
    "heading1": "Defnyddio Prosiectau Seilwaith Cenedlaethol",
    "paragraph1": "Rydych yn cytuno i ddefnyddio {{-link}} at ddibenion cyfreithlon yn unig. Mae'n rhaid i chi hefyd ei ddefnyddio mewn ffordd nad yw'n torri hawliau unrhyw un arall i ddefnyddio'r safle hwn nac yn rhwystro na chyfyngu ar ei ddefnydd a'i fwynhad ohono.",
    "paragraph2": "Rydym yn diweddaru {{-link}} drwy'r amser. Gallwn newid neu ddileu cynnwys unrhyw bryd heb roi rhybudd.",
  },
  "usingNIPContent": {
    "heading1": "Defnyddio cynnwys Prosiectau Seilwaith Cenedlaethol",
    "paragraph1": "Mae rhywfaint o gynnwys wedi'i eithrio o'r OGL – gwiriwch y {{-link}}.",
    "paragraph10": "Bydd y fersiwn fwyaf cyfredol o'n cynnwys ar {{-link}} bob amser.",
    "paragraph1LinkText": "rhestr eithriadau",
    "paragraph2": "Mae logos ac arwyddluniau adrannol wedi'u heithrio o'r OGL hefyd, heblaw pan fyddant yn rhan annatod o ddogfen neu set ddata.",
    "paragraph3": "Os oes unrhyw gynnwys nad yw'n destun amddiffyniad hawlfraint y Goron neu'n cael ei gyhoeddi o dan yr OGL, byddwn yn cydnabod yr awdur neu'r deiliad hawlfraint, fel arfer.",
    "paragraph4": "Nid ydym yn awdurdodi defnyddio neu ailgyhoeddi dogfennau trydydd parti heb ganiatâd y deiliad eiddo deallusol gwreiddiol.",
    "paragraph5": "Mae hyn yn cynnwys yr holl ddeunydd nad yw {{-link}} yn berchen arno, gan gynnwys hwnnw a grëwyd gan ymgeiswyr, partïon â buddiant neu grwpiau â diddordeb.",
    "paragraph6": "Os oes gennych gwestiynau am ailgyhoeddi unrhyw gynnwys ar ein gwasanaeth, {{-link}}.",
    "paragraph7": "{{-link}} os hoffech atgynhyrchu darn o gynnwys ond nad ydych yn siŵr a yw'n destun hawlfraint y Goron neu'r OGL.",
    "paragraph8": "Rydym yn sicrhau bod y rhan fwyaf o'r cynnwys ar {{-link}} ar gael trwy ffrydiau i'w defnyddio gan wefannau a chymwysiadau eraill. Nid ein cynhyrchion ni yw'r gwefannau a'r cymwysiadau sy'n defnyddio ein ffrydiau, ac mae'n bosibl y byddant yn defnyddio fersiynau o'n cynnwys sydd wedi cael eu golygu a'u storio i'w defnyddio'n ddiweddarach ('eu storio').",
    "paragraph9": "Nid ydym yn rhoi unrhyw sicrwydd, amodau na gwarantau ynglŷn â chywirdeb neu gyflawnrwydd unrhyw gynnwys a ddefnyddir gan y cynhyrchion hyn. Nid ydym yn atebol am unrhyw golled neu ddifrod a allai ddeillio o'ch defnydd o'r cynhyrchion hyn.",
    "phrase1": "Mae'r rhan fwyaf o'r cynnwys ar {{-link}}",
    "phrase2": "yn destun {{-link}}",
    "phrase2LinkText": "amddiffyniad hawlfraint y Goron",
    "phrase3": "a chaiff ei gyhoeddi o dan y {{-link}} (OGL).",
    "phrase3LinkText": "Drwydded Llywodraeth Agored",
  },
  "virusProtection": {
    "heading1": "Diogelu rhag feirysau",
    "paragraph1": "Rydym yn gwneud pob ymdrech i wirio a phrofi {{-link}} am feirysau ar bob cam cynhyrchu.",
    "paragraph2": "Mae'n rhaid i chi wneud yn siŵr nad yw'r ffordd rydych yn defnyddio {{-link}} yn eich gwneud yn agored i risg feirysau, cod cyfrifiadurol maleisus neu fathau eraill o ymyrraeth sy'n gallu niweidio'ch system gyfrifiadurol.",
    "paragraph3": "Nid ydym yn gyfrifol am unrhyw golled, tarfu neu ddifrod i'ch data neu'ch system gyfrifiadurol a allai ddigwydd pan fyddwch yn defnyddio {{-link}}.",
  },
  "whoWeAre": {
    "heading1": "Pwy ydym ni",
    "paragraph1": "Mae {{-link}} yn cael ei weithredu gan yr Arolygiaeth Gynllunio.",
    "paragraph2": "Mae'r Arolygiaeth Gynllunio yn asiantaeth y llywodraeth.",
    "paragraph3": "Os oes gennych gwestiynau am ein gwasanaeth, {{-link}}.",
  },
}
`;
